### DISCUSSION TITLE
Will be added example run with npm ?

### DISCUSSION THREAD

**Discussion by wildwind123** [histoire-dev/histoire#114] [Q&A] [answered]
Like for pnpm ? 
[quick start](https://github.com/histoire-dev/histoire#quick-start)

**Comment by kecrily** [comment]
It is only one `p` away from `npm`

**Reply by Akryum** [reply]
😹

**Comment by Akryum** [comment, marked as answer]
It should work with npm, if it doesn't, please post a bug report. Thank you!
